Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -182,6 +187,16 @@ void UpdateTextColor()
 			_textColorSwitcher?.UpdateTextColor(Control, Element.TextColor);
 		}
 
+		void UpdateDisabledTextColor()
+		{
+			int[][] states = { new[] { AResource.Attribute.StateEnabled }, new[] { -AResource.Attribute.StateEnabled } };
+			int[] colors = { Control.TextColors.GetColorForState(states[0], Element.TextColor.ToAndroid()), Element.DisabledTextColor.ToAndroid().ToArgb() };
+
+			var colorStateList = new ColorStateList(states, colors);
+
+			Control.SetTextColor(colorStateList);
```

Comment:
Looks like this method updates not only the `DisabledTextColor`, but also the `TextColor`.

Reply:
I'm retrieving the current `TextColor`, and I set it as the `Enabled` text color. So in fact it updates it, but it doesn't set a new value, it just updates it to the same value. I don't know if I can do it another way. :-)